app/views/finance/ecf/duplicates/compare/_participant_declaration_summary_list.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= govuk_summary_list do |summary_list|
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "ID" }
+    row.value  { participant_declaration.id }
+  end
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "Type" }
+    row.value  { participant_declaration.declaration_type }
+  end
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "State" }
+    row.value  { participant_declaration.state }
+  end
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "Lead provider" }
+    row.value  { participant_declaration.cpd_lead_provider.name }
+  end
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "Declaration date" }
+    row.value { participant_declaration.declaration_date.to_s(:short) }
+  end
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "Course" }
+    row.value { participant_declaration.course_identifier }
+  end
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "Sparsity uplift" }
+    row.value { participant_declaration.course_identifier }
```

Comment:
this and the row below aren't included in the query, do we want to remove it for now, or add to the query?